CentOS 7 build failures

## Bug Report

On new PRs, CentOS 7 CI workflow is failing with the following error:

```
WARNING: No output specified with docker-container driver. Build result will only remain in the build cache. To push result image into registry use --push or to load image into docker use --load
Dockerfile.centos7:28
--------------------
  26 |         -DFLB_OUT_PGSQL=On ../
  27 |     
  28 | >>> RUN make -j "$(getconf _NPROCESSORS_ONLN)"
  29 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c make -j \"$(getconf _NPROCESSORS_ONLN)\"" did not complete successfully: exit code: 2
Error: buildx failed with: ERROR: failed to solve: process "/bin/sh -c make -j \"$(getconf _NPROCESSORS_ONLN)\"" did not complete successfully: exit code: 2
```

ref: https://github.com/fluent/fluent-bit/actions/runs/8355812140/job/22871635010?pr=8605

It also looks like this fails on other CentOS targets so may be a more general problem:
- centos/8
```
[ 36%] Building C object lib/ctraces/src/CMakeFiles/ctraces-static.dir/ctr_id.c.o
In file included from sql-parser.y:16:
/tmp/fluent-bit/build/plugins/processor_sql/parser/processor-sql-parser_lex.h:213:41: error: expected identifier or '(' before numeric constant
 #define yywrap(yyscanner) (/*CONSTCOND*/1)
                                         ^
sql-parser.y:29:5: note: in expansion of macro 'yywrap'
make[2]: *** [plugins/processor_sql/parser/CMakeFiles/processor-sql-parser.dir/build.make:133: plugins/processor_sql/parser/CMakeFiles/processor-sql-parser.dir/processor-sql_parser.c.o] Error 1
```

--- a/plugins/processor_sql/parser/sql-parser.y
+++ b/plugins/processor_sql/parser/sql-parser.y
@@ -26,11 +26,6 @@ void yyerror (struct sql_query *query, void *scanner, const char *str)
     fprintf(stderr, "error: %s\n", str);
 }
 
-int yywrap(struct sql_query *query)
-{
-    return 1;
-}
-
 %} /* EOF C code */
 
 

